packages/desktop_drop/example/android/build.gradle

```diff
@@ -24,6 +24,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
This change has been automated too by the Flutter CLI, to migrate from deprecated Gradle Groovy syntax.